WalletWasabi/Helpers/HttpMessageHelper.cs

```diff
@@ -117,12 +118,10 @@ private static async Task<string> ReadCRLFLineAsync(Stream stream, Encoding enco
 				}
 				bab.Append((byte)ch);
 			}
-			if (bab.Length > 0)
-			{
-				return bab.ToString(encoding);
-			}
 
-			return null;
+			return bab.Length > 0
+				? bab.ToString(encoding)
+				: null;
```

Comment:
I don't like it.

Reply:
Should it be inlined or should it be removed?

Reply:
IDK, @molnard likes it so, I guess I merge with this.